src/collectors/mogilefs/mogilefs.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+
+"""
+Collect statistics from Mogilefs
+
+#### Dependencies
+
+ * telnetlib
+ * time
+
+
+"""
+import diamond.collector
+import telnetlib
+import time
+
+
+class MogstatsCollector(diamond.collector.Collector):
+
+    def get_default_config_help(self):
+        config_help = super(MogstatsCollector, self).get_default_config_help()
+        config_help.update({
+        })
+        return config_help
+
+    def get_default_config(self):
+        """
+        Returns the default collector settings
+        """
+        config = super(MogstatsCollector, self).get_default_config()
+        config.update({
+            'path':     'mogilefsd.stats'
```

Comment:
what is at the `mogilefsd` path? if nothing, this should probably go there and not at a sub path